cpp_utils/BLEServer.cpp

```diff
@@ -115,6 +115,21 @@ BLEService* BLEServer::getServiceByUUID(BLEUUID uuid) {
 }
 
 
+/**
+ * @brief Returns the amount of services registered to this server
+ * @return The amount of registered services
+ */
+int BLEServer::getServiceCount() {
+	int count = 0;
+	if(m_serviceMap.getFirst() == nullptr) return 0;
+	while(m_serviceMap.getNext() != nullptr){
```

Comment:
m_serviceMap is std::map and has size() function to get count of elements.
https://en.cppreference.com/w/cpp/container/map/size

Also it will only return services that we are storing, without 2 generic services.

Reply:
ah, ok didn't know about that. Thank you for clarifying. I didn't know that there is are services which are not stored...

Reply:
> Also it will only return services that we are storing, without 2 generic services.
Are there always two generic services? Or is it possible to check if these services exist? I will update my code then...

Reply:
Always 2 generic services, it is bluetooth specs mandatory. Its 0x1800 and 0x1801